### DISCUSSION TITLE
PDK should not error out on validation of non-converted modules

### DISCUSSION THREAD

**Discussion by trevor-vaughan** [puppetlabs/pdk#1012] [Feature Request]
**Describe the bug**

When running `pdk validate` on an unconverted module, I get an error stating that the command cannot be run without converting the module first.

This seems to be a bug since the format of a given module's helper files is orthogonal whether or not is is valid.

**To Reproduce**

Go to any unconverted module, run `pdk validate` watch it tell you that you need to convert your module before trying to validate it.

**Expected behavior**

The module would be validated.

**Comment by natemccurdy** [comment]
Probably related to, or at least similar to, my request here: puppetlabs/pdk#980

**Comment by trevor-vaughan** [comment]
@natemccurdy Pretty much the exact same thing, yes.
